### DISCUSSION TITLE
TOML compatible config file

### DISCUSSION THREAD

**Discussion by John-Appleseed** [Kozea/Radicale#1372] [Ideas]
Radicale's config file is very close to the TOML config format. 
For TOML parsers to work with Radicale configs:
- [ ] Allow strings with quotes. ex `hosts = "127.0.0.1:5232"`

TOML Config Test

```
# Install dasel
go install github.com/tomwright/dasel/cmd/dasel@master
```
```
# Write config options with dasel to radicale/config
dasel put -p toml string -f "radicale/config" "server.hosts" "127.0.0.1:5232"
```

**Comment by pbiering** [comment]
can you please create a PR for that?
